packages/origin-ui-core/src/components/Account/UserProfile/ChangePassword.tsx

```diff
@@ -0,0 +1,165 @@
+        }
+    }
+
+    return (
+        <Formik
+            initialValues={INITIAL_VALUES}
+            validateOnMount={true}
+            onSubmit={submitForm}
+            validate={ValidationHandler}
+        >
+            {(formikProps: FormikProps<typeof INITIAL_VALUES>) => {
+                const { isSubmitting, touched } = formikProps;
+                const fieldDisabled = isSubmitting || !isEditing;
+                return (
+                    <Form translate="no">
+                        <Paper className={classes.container}>
+                            <Typography variant="h5">
+                                {t('user.profile.subtitle.security')}
+                            </Typography>
+                            <Grid container spacing={3}>
+                                <Grid item xs={4}>
+                                    <FormInput
+                                        label="Current Password"
+                                        property="currentPassword"
+                                        disabled={fieldDisabled}
+                                        className="mt-3"
+                                        type="password"
+                                        required
+                                    />
+                                </Grid>
+                                <Grid item xs={4}>
+                                    <FormInput
+                                        label="New Password"
+                                        property="newPassword"
+                                        disabled={fieldDisabled}
+                                        className="mt-3"
+                                        type="password"
+                                        required
+                                    />
+                                </Grid>
+                                <Grid item xs={4}>
+                                    <FormInput
+                                        label="Confirm Password"
+                                        property="newPasswordConfirm"
+                                        disabled={fieldDisabled}
+                                        className="mt-3"
+                                        type="password"
+                                        required
+                                    />
+                                </Grid>
+                            </Grid>
+
+                            {isEditing && (
+                                <Button
+                                    style={{ marginRight: 10 }}
+                                    type="button"
+                                    variant="contained"
+                                    color="primary"
+                                    className="mt-3 right"
+                                    disabled={!(touched.currentPassword || touched.newPassword)}
+                                    onClick={async () => {
```

Comment:
When clicking Save in ChangePassword the fields don't get empty. Save and Cancel buttons also don't hide.

Reply:
fixed, now after fields should be cleared after saving or canceling